doc/lang_tooltip_dialog.md

```diff
@@ -39,3 +39,27 @@ This is the list of accepted and supported locales in your game.
 The game will crash if you define a locale that's not accepted, so don't add unsupported
 locales to your game code or define an unsupported default ;)
 
+## Dialog overlay
+
+To keep lag to a minimum when displaying dialog text, the `dialog player` is a `ResourcePreloader`,
+which packs the scene that displays your text.
+
+The default is located in `device/ui/dialog_default.tscn`. If you wish to customize its appearance,
+make a copy like `device/game/ui/dialog.tscn`.
+
+You may experience pain when trying to adjust the `text` node to your liking. Godot isn't very
+intuitive when it comes to expanding the `RichTextLabel`. Hopefully this will behave more like a
+simple `Label` in the future. In the mean while, Escoria relies heavily on the current type, as it
+should to enable italic and bold font.
+
+The setting `escoria/platform/dialog_force_centered` will center your dialog text if your
+game is like SCUMM games. If you use avatars and big boxes for dialog, you will probably want to
+leave this disabled.
+
+Last you will need to - sadly - edit the Escoria code itself. `device/ui/dialog_player.tscn` is
+the file where this is defined, so you can edit it with the Godot editor or open it up in a text
+editor and update your path to `device/game/ui/dialog.tscn`.
```

Comment:
Ok, your "sadly" made me think you meant there was a bug or shortcoming in Escoria. I don't think it's a problem really, only you can't easily see what those scenes look like. How would you have prefered to configure dialogue scenes?

Reply:
Here I wish I had a perfect solution :( 

There is nothing in the scene to see, right? It's just a placeholder for the script, essentially, and a means of preloading `dialog_default.tscn` which is just a default.

Most of the scenes are structured by adding nodes to them, but in this case there is no adding a node, there is editing that preload.

I believe @StraToN pointed out that this is a necessary optimization. I understood the scene contents can't be altered easily through ProjectSettings. So we're kinda stuck.

Maybe if #5 gets done, a simple scene like this could be created in `OS.File.write()` fashion with a dialog UI from the user.

Or is there a good way to replace that `ext_resource` in the Escoria startup code?

Reply:
(Unless there is a way of replacing that, it is a shortcoming in Escoria, or maybe even Godot, because the coupling's too tight)

Reply:
An alternative that I suggested earlier today for the hud elements would be to cache everything under a given file path, eg `device/demo/ui/`. Using that method, maybe the dialog scenes could be children of a regular scene, if that caching provides the necessary optimization. I don't know how easy it would be to benchmark though, except running both implementations on a low-end device and seeing if there's a noticeable difference.

Reply:
In any case, I think it's safer to recommend that users change dialogue scenes with the file picker in the ResourcePreloader, since there's less chance of messing things up that way.

Reply:
> (Unless there is a way of replacing that, it is a shortcoming in Escoria, or maybe even Godot, because the coupling's too tight)

Is it tighter than any of the other UI scenes though? If you wanted to, you could use it to hold separate scenes for the demo and your own game without problems. The dialogue actions have parameters to decide which scene is used. The main/only problem the way I see it is that there's no easy way to see what the dialog scenes look like from the ResourcePreloader.

Reply:
When the hud and game were split, the coupling loosened on their part enough that I could do a lot of stuff at https://github.com/mjtorn/escoria/tree/devel/device so that's something.

I also don't know about benchmarking... I have usually learned to trust the OS with these matters, but if mobile platforms don't eg. have a file cache despite having slow I/O, there's the case that blows the argument out of the water.

And lol, sorry, I think I documented something completely wrong about the default. I didn't remember that `say` took an argument there, so the documentation recommends a wrong name. I hope I can sneak away from the family for long enough to update this PR so we don't end up blocking for very long.

Also, whatever you see fit as the way to cache or organize files is probably fine. Just tell me when I can test something, or go ahead and merge a new directory structure and I'll happily rebase.

Reply:
I did actually get a moment to update the documentation. I think it's better now. It would be even better with instructions on how to cache more dialog types, but I think the document will require some overhaul later anyways.